### PR TITLE
Apply the new device crate changes

### PR DESCRIPTION
**Pull request by zakorgy**
## Refactor get_included_service(s) methods

- [x] `./mach build -d` does not report any errors

### CODE REVIEW

File: components/net/bluetooth_thread.rs [hunk 1]
```diff
@@ -633,11 +633,22 @@ impl BluetoothManager {
             Some(a) => a,
             None => return drop(sender.send(Err(BluetoothError::Type(ADAPTER_ERROR.to_string())))),
         };
+        let device;
```
Comment (dati91) [inline components/net/bluetooth_thread.rs]: I don't get why would you use this syntax :/
why not `let device = ...` in line 642?

File: components/net/bluetooth_thread.rs [hunk 2]
```diff
@@ -660,11 +671,22 @@ impl BluetoothManager {
...
+        {
+            let device_id = match self.service_to_device.get(service_id.as_str()) {
+                Some(id) => id,
+                None => return drop(sender.send(Err(BluetoothError::NotFound))),
+            };
+            device = match self.cached_devices.get(device_id) {
```
Comment (dati91) [inline components/net/bluetooth_thread.rs]: Same as above.

**Review by dati91 — CHANGES_REQUESTED** [review]

**Comment by dati91** [comment]
You should update the cargo.lock because there was some changes with the blurmock crate.

**Comment by zakorgy** [comment]
I made the requested changes, and updated the Cargo.lock file. r? @dati91

**Pull request closed without merging**